config/routes.rb

```diff
@@ -5,6 +5,7 @@
   # You can have the root of your site routed with "root"
   root 'static_pages#index'
   get 'movies/:year' => 'movies#show'
+  get 'songs/:year' => 'songlists#show'
```

Comment:
Maybe the name of your route should be the same as the corresponding controller. Otherwise, it may become confusing.